src/sinol_make/helpers/package_util.py

```diff
@@ -61,39 +62,65 @@ def get_executable_path(solution: str) -> str:
     return os.path.join(os.getcwd(), 'cache', 'executables', get_executable(solution))
 
 
-def get_time_limit(test_path, config, args=None):
+def get_file_lang(file_path):
+    return os.path.splitext(file_path)[1][1:].lower()
+
+
+class LimitTypes(Enum):
+    TIME_LIMIT = 1
+    MEMORY_LIMIT = 2
+
+
+def _get_limit_from_dict(dict: Dict[str, Any], limit_type: LimitTypes, test_id: str, test_group: str):
+    if limit_type == LimitTypes.TIME_LIMIT:
+        limit_name = "time_limit"
+        plural_limit_name = "time_limits"
+    elif limit_type == LimitTypes.MEMORY_LIMIT:
+        limit_name = "memory_limit"
+        plural_limit_name = "memory_limits"
+    else:
+        raise ValueError("Invalid limit type.")
```

Comment:
I believe there should be a `match` statement in Python, which would work on enums?

Reply:
`match` is only supported from python 3.10